### ISSUE TITLE
HTML Response Incorrect

### ISSUE THREAD

**Issue by joe-cole1** [olizimmermann/pangolin_rule_updater#2]
Let me preface this by saying the Rules update portion seems to be working just fine.

If I visit the site and an update IS required, I get this HTML response, with the logs below:

```
IP Update Trigger
An update was not triggered.
```

```
<socket.socket fd=4, family=2, type=1, proto=0, laddr=('172.18.0.8', 8989), raddr=('172.18.0.7', 35672)>
[info] Incoming request: auth.url.com/url
[info] Incoming request from: xxx.xxx.xxx.xxx (my actual IP)
[info] IP address changed: xxx.xxx.xxx.xxx != 1.1.1.1
[pangolin] Updated rule 17 to xxx.xxx.xxx.xxx
```

If, however, the IP is already set, I get the following response with the logs below:

```Internal Server Error```

```
<socket.socket fd=4, family=2, type=1, proto=0, laddr=('172.18.0.8', 8989), raddr=('172.18.0.7', 39990)>
[info] Incoming request: auth.url.com/url
[info] Incoming request from: xxx.xxx.xxx.xxx
[info] IP address unchanged: xxx.xxx.xxx.xxx == xxx.xxx.xxx.xxx
```

So it's working fine on the backend! Just the HTML frontend is not quite right.

**Comment by olizimmermann** [comment]
Hey! You wanna try it again? :) @joe-cole1

**Comment by joe-cole1** [comment]
Nice... fixed!

**Comment by olizimmermann** [comment]
will merge it then. in case you pull it from main again, I did comment out the ports in the docker compose file - it's cleaner not to expose anything if you not need it :)

**Comment by olizimmermann** [comment]
Just tagging you again to see my response: @joe-cole1

**Comment by joe-cole1** [comment]
Sounds great!
